js/assessments/paragraphTooShortAssessment.js

```diff
@@ -3,34 +3,35 @@ var fixFloatingPoint = require( "../helpers/fixFloatingPoint.js" );
 var isParagraphTooShort = require( "../helpers/isValueTooShort" );
 var filter = require( "lodash/filter" );
 
+var recommendedValue = 40;
```

Comment:
I would like this when it's documented. I guess `recommendedValue` contains the value of words in a sentence. It is now too generic.